packages/runtime-core/src/renderer.ts

```diff
@@ -2215,6 +2203,14 @@ function baseCreateRenderer(
     hostRemove(end)
   }
 
+  const getFirstElement = (cur: RendererNode, end: RendererNode) => {
+    while (cur.nodeType !== Node.ELEMENT_NODE && cur !== end) {
```

Comment:
should use `DOMNodeTypes.ELEMENT` instead of `Node.ELEMENT_NODE` due to `runtime-core` is cross-platform

Reply:
Done.